ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -78,6 +78,21 @@ tests:
     - ref: cucushift-upgrade-healthcheck
     - ref: cucushift-ota-postupgrade
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: aws-ipi-efa-pg-f28
```

Comment:
Suggest to add some keyword in the job name to indicated it is covering tolatest upgrade option.